src/main/resources/application.yaml

```diff
@@ -37,6 +37,13 @@ spring:
 idam:
   api:
     url: ${IDAM_URL:http://localhost:5000}
+  client:
+    id: aac_manage_case_assignment
+    secret: ${IDAM_CLIENT_SECRET:AAAAAAAAAAAAAAAA}
+    redirect_uri: ${OAUTH2_REDIRECT_URI:http://localhost:3451/oauth2redirect}
```

Comment:
you might have to define OAUTH2_REDIRECT_URI in helm values.yml with matching idam value. Or you can hardcode here itself as it's same fo all environments in MCA. ( we've provided in the client config as Idam mandates that but mca never redirect to any other link)